xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -714,6 +714,12 @@ bool CGUIWindowVideoBase::OnItemInfo(int iItem)
     else
       strDir = URIUtils::GetDirectory(item->GetPath());
 
+    if (URIUtils::IsUPnP(strDir) && item->IsVideo())
+    {
+      CGUIDialogVideoInfo::ShowFor(*item);
```

Comment:
Yes, I see this now.  I've reverted the commit for this and added a new commit which does not call CGUIDialogVideoInfo::ShowFor but instead, allows items that are from a UPNP source to continue through CGUIWindowVideoBase::OnItemInfo instead of returning false.